libs/npm/core/src/utils/getCookie.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Get the value for the specified 'key' in the cookie.
+ * @param key The key name.
+ * @returns The value if exists.
+ */
+export const getCookie = (key: string) => {
+  console.debug('get', document.cookie, window.location.pathname);
+  var b = document.cookie.match('(^|;)\\s*' + key + '\\s*=\\s*([^;]+)');
```

Comment:
Isn't working on Chrome.
On FF it will return a cookie for the wrong path for some reason!

Reply:
These cookies seem to be a pain - think you're right that local storage would be best